doc/win-dev.ps1

```diff
@@ -83,6 +82,7 @@ if (-not $Env:GITHUB_ACTIONS) {
         git `
         cmake `
         winflexbison3 `
+        netfx-4.6-devpack `
```

Comment:
Is this a replacement for `visualstudio${VsVersion}-workload-netcoretools`? This is the only non-obvious change here, so a few more words on it would be nice.